src/controller/CHIPDevice.h

```diff
@@ -396,7 +403,8 @@ class DLL_EXPORT Device : public Messaging::ExchangeDelegate, public SessionEsta
 
     Inet::InetLayer * mInetLayer = nullptr;
 
-    bool mActive           = false;
+    bool mActive = false;
+    // Should be set to true only if the commissioner provides a CSRNonce during commissioning flow.
```

Comment:
nit: delete this comment. possibly left over from the deleted change in this PR.

Reply:
done